src/components/Onboarding/Screens/Apps/VotingSettings/index.js

```diff
@@ -276,15 +240,70 @@ function VotingSettings() {
         value={voteMinAcceptanceQuorum}
         onChange={handleQuorumChange}
       />
-      <div>
-        <Carousel
-          itemWidth={80 * GU}
-          itemHeight={38 * GU}
-          itemSpacing={2 * GU}
-          items={carouselItems}
-          onItemSelected={handleItemSelected}
-        />
-      </div>
+      <DurationFields
+        label={
+          <Fragment>
+            Voting Duration
+            <Help hint="What is Vote Duration?">
+              <strong>Vote Duration</strong> is the length of time that the vote
+              will be open for participation. For example, if the Vote Duration
+              is set to 24 hours, then tokenholders have 24 hours to participate
+              in the vote.
+            </Help>
+          </Fragment>
+        }
+        duration={voteDuration}
+        onUpdate={handleDurationChange}
+      />
+      <DurationFields
+        label={
+          <Fragment>
+            Execution Delay
+            <Help hint="What is Execution Delay?">
+              <strong>Execution Delay</strong> is the required amount of time
+              after a proposal passes for the proposal to be executed. This
+              allows everyone to react to the outcome and make decisions before
+              the effects of the vote are realised.
+            </Help>
+          </Fragment>
+        }
+        duration={voteExecutionDelay}
+        onUpdate={handleExecutionDelayPeriodChange}
+      />
+      <DurationFields
+        label={
+          <Fragment>
+            Quite Ending Period
+            <Help hint="What is Quite Ending Period?">
+              <strong>Quite Ending Period</strong> is the duration before the
+              end of a vote to detect non-quiet endings. Non-quiet endings are
+              endings which involve a late swing in the vote.
+            </Help>
+          </Fragment>
+        }
+        duration={voteQuietEndingPeriod}
+        onUpdate={handleQuiteEndingPeriodChange}
+      />
+      <Button
+        size="mini"
+        css={`
+          align-self: flex-end;
+          float: right;
+          margin-bottom: ${5 * GU}px;
+        `}
+        label="Advanced..."
+        onClick={() => setOpenSettingsModal(true)}
+      />
```

Comment:
I would avoid using `float` for layout and the `align-self` style is not doing anything in this case. We can wrap the button with a div and apply styles to it.

Also, what do you think of using a `handleOpenModal` callback as we have for closing the modal?



```suggestion 
      <div 
         css={`  
            display: flex;
            align-items: flex-end; 
            margin-bottom: ${5 * GU}px;
          `}
       >
         <Button
           size="mini"
           label="Advanced..."
           onClick={handleOpenModal}
         />
     </div>
```